_security-analytics/api-tools/rule-api.md

```diff
@@ -11,7 +11,7 @@ The following APIs can be used for a number of tasks related to rules, from sear
 
 ## Create Custom Rule
 
-The Create custom rule API uses Sigma security rule formatting to create a custom rule. For information on how to write a rule in Sigma format, see information provided at [Sigma's GitHub repository](https://github.com/SigmaHQ/sigma).
+The Create custom rule API uses Sigma security rule formatting to create a custom rule. For information about how to write a rule in Sigma format, see information provided at [Sigma's GitHub repository](https://github.com/SigmaHQ/sigma).
```

Comment:
Create custom rule API or Create Custom Rule API?

Reply:
I like the second. Thanks for pointing out.